jsettlers.tools/src/main/java/jsettlers/buildingcreator/editor/GraphicsMovable.java

```diff
@@ -0,0 +1,127 @@
+/*
+ */
+package jsettlers.buildingcreator.editor;
+
+import jsettlers.common.material.EMaterialType;
+import jsettlers.common.movable.EDirection;
+import jsettlers.common.movable.EEffectType;
+import jsettlers.common.movable.EMovableAction;
+import jsettlers.common.movable.EMovableType;
+import jsettlers.common.movable.IGraphicsMovable;
+import jsettlers.common.player.IPlayer;
+import jsettlers.common.position.ShortPoint2D;
+import jsettlers.common.selectable.ESelectionType;
+
+/**
+ *
+ * @author hiran
+ */
+public class GraphicsMovable implements IGraphicsMovable {
```

Comment:
I don't want to nitpick but I kind of dislike this name because it doesn't specify that this is only a dummy class and not something that should be used.
Maybe call it PseudoMovable, DummyMovable or BuildingCreatorMovable?
I think FakeMovable is taken